### PR TITLE
Add graph panel popup

### PR DESCRIPTION
**Pull request by kevinjcash**
Tweak sizing

Remove console log

make popup scale to screen and be unresizable

Remove commented out maxWidth

### CODE REVIEW

File: Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js [hunk 1]
```diff
@@ -182,6 +187,11 @@
                     xtype: 'toolbar',
                     dock: 'top',
                     items: ['->',{
+                        text: 'Open in New Tab',
```
Comment (jhanson) [inline Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js]: all user facing text needs to be wrapped in an "_t" method for translations.

File: Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js [hunk 2]
```diff
@@ -168,7 +174,6 @@
             config.buttonId = Ext.id();
             config = Ext.applyIf(config||{}, {
                 html: this.graphTemplate.apply(config),
-                maxWidth: 800,
```
Comment (jhanson) [inline Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js]: This makes the component graphs look terrible, can we explicitly set this to null on the pop up graph? 
screenshot 2015-06-11 07 28 07

Comment (kevinjcash) [inline Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js]: Reinstate the maxWidth for the normal panels?

Comment (jhanson) [inline Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js]: I think so

File: Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js [hunk 3]
```diff
@@ -182,6 +188,11 @@
                     xtype: 'toolbar',
                     dock: 'top',
                     items: ['->',{
+                        text: _t('Open in New Tab'),
+                        handler: Ext.bind(function(btn, e) {
```
Comment (jhanson) [inline Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js]: This isn't necessary but it can be rewritten as 

Ext.bind(this.newTab, this);

**Pull request merged**